src/includes/getting-started-install/android.okhttp.mdx

```diff
@@ -0,0 +1,12 @@
+<Note>
+
+This is a preview API, available in `4.4.0-alpha.1`. Features available in a preview API are still in-progress and may have bugs. We recognize the irony. The API may be renamed, changed, or removed in a future version.
```

Comment:
`l` Maybe we can move this to the top of the `getting-started-primer`, so it's the first sentence on the page. Having it in the middle looks a bit weird to me.

<img width="617" alt="Screenshot 2021-03-26 at 10 18 01" src="https://user-images.githubusercontent.com/2443292/112610042-8c5bff80-8e1c-11eb-922d-27b7585b7236.png">

Reply:
makes sense